src/data/equiv/algebra.lean

```diff
@@ -609,6 +609,20 @@ begin
   { exact congr_arg equiv.inv_fun h₁ }
 end
 
+/-- If two rings are isomorphic, and the second is an integral domain, then so is the first. -/
+protected lemma is_integral_domain {A : Type u} [comm_ring A]
```

Comment:
Isn't it strange to have a function converting `integral_domain B` to `is_integral_domain A`? I would have expected the natural API to convert `is_integral_domain B` to `is_integral_domain A`, and `integral_domain B` to `integral_domain A`.

Reply:
This is now fixed.